### PR TITLE
Add Fall Fest content

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Closes https://github.com/Qiskit/qiskit.org/issues/2787

## Branch preview
[Preview link](https://qiskit-org-pr-2793.dcq4xc5i083.us-south.codeengine.appdomain.cloud/events/fall-fest)

## Implementation details

- added content for Qiskit Fall Fest
- added assets for each event (from Brian)

## Screenshots

https://user-images.githubusercontent.com/6276074/192562256-6be96dcc-ee0e-4cb0-b91e-05be9d87b50e.mov

### CODE REVIEW

**Review by vabarbosa — COMMENTED** [review]
@techtolentino one question does the `package-lock.json` need updating?

File: components/events/fall-fest/UniversityDirectorySection.vue [hunk 1]
```diff
@@ -38,111 +43,331 @@ import { GeneralLink } from '~/constants/appLinks'
...
+      }
+    },
+    {
+      title: 'University of Chicago',
+      image: '/images/events/fall-fest/illinois.png',
+      startDate: 'October 10, 2022',
+      detail: 'In Person',
+      cta: {
+        label: 'Learn more here',
+        url: '',
```
Comment (vabarbosa) [inline components/events/fall-fest/UniversityDirectorySection.vue:248]: is there no URL for this one?

Comment (techtolentino) [reply components/events/fall-fest/UniversityDirectorySection.vue:248]: @vabarbosa 

I can revert the package lock, good call out.

For empty `url` values, yes -- some schools are still in the process of getting the information to Brian. We suspect that going _live_ will be one way to give them more incentive get their info to him sooner.

🤞🏼

Comment (vabarbosa) [reply components/events/fall-fest/UniversityDirectorySection.vue:248]: cool. will approve

**Review by vabarbosa — APPROVED** [review]
@techtolentino thank you!

**Pull request merged**